Kickstarter-iOS/Features/PledgeView/Views/EstimatedShippingCheckoutView.swift

```diff
@@ -0,0 +1,54 @@
+import Library
+import SwiftUI
+
+struct EstimatedShippingCheckoutView: View {
+  let estimatedCost: String
+  let aboutConversion: String
+
+  private enum Constants {
+    /// Spacing & Padding
+    public static let defaultSpacing = Styles.grid(1)
+    public static let verticalPadding = Styles.grid(2)
+    public static let horizontalPadding = Styles.grid(4)
+    public static let maxViewHeight = 125.0
```

Comment:
How does this max height work for accessibility large sizes? I'm concerned that the text will get truncated enough that it won't be usable.

Reply:
Ah good call. Removed. Here's how it looks with large accessibility font sizes 
![Simulator Screen Recording - iPhone 15 Pro - 2024-08-26 at 08 58 36](https://github.com/user-attachments/assets/555e365e-d7db-400f-a68d-946d2d8b73a5)